android/app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
     <!-- Base application theme. -->
-    <!-- Set the base theme to light for the entire app -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    <!-- Support light and dark mode in the entire app -->
+    <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
         <!-- Additional customizations can be added here -->
         <item name="android:enforceNavigationBarContrast">false</item> <!-- Optional: Disabling enforced contrast for navigation bar -->
         <item name="android:forceDarkAllowed">false</item> <!-- Prevent the system from applying automatic dark mode transformations -->
```

Comment:
what about this one?

Reply:
Is seems to be about the automatic algorithmic dark theme that can be enabled by Android. So I think is it better to let it forced at false. We only want our dark theme and not algorithmic dark theme.

https://developer.android.com/develop/ui/views/theming/darktheme#force-dark
https://medium.com/@juniaporto/android-how-to-fix-forced-dark-mode-in-xiaomi-devices-fcd39f037c26